### PR TITLE
Remove nulls from UncheckedProc #885

### PR DESCRIPTION
**Pull request by victornoel**
This is for #885

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1029](https://github.com/yegor256/cactoos/issues/1029) is now in scope, role is `REV`

File: src/main/java/org/cactoos/func/UncheckedProc.java [hunk 1]
```diff
@@ -53,7 +52,11 @@ public UncheckedProc(final Proc<X> prc) {
 
     @Override
     public void exec(final X input) {
-        new UncheckedFunc<>(new FuncOf<>(this.proc, null)).apply(input);
+        try {
```
[the remark this answers is not in the document]
Comment (victornoel) [reply src/main/java/org/cactoos/func/UncheckedProc.java]: @fabriciofx there already are tests for this class

[the remark this answers is not in the document]
Comment (fabriciofx) [reply src/main/java/org/cactoos/func/UncheckedProc.java]: @victornoel Right. Thanks!

**Comment by fabriciofx** [comment]
@victornoel This PR seems fine. Thanks! @llorllale Can you merge it now?

**Comment by fabriciofx** [comment]
@rultor merge

File: src/main/java/org/cactoos/func/UncheckedProc.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (llorllale) [reply src/main/java/org/cactoos/func/UncheckedProc.java]: @victornoel you can use `CheckedScalar` here

[the remark this answers is not in the document]
Comment (victornoel) [reply src/main/java/org/cactoos/func/UncheckedProc.java]: @llorllale if I use `CheckedScalar` then I will need to specify an object to be returned, and then I will put `null` :) `FuncOf` does use one of the `Scalar` class for example

**Comment by llorllale** [comment]
@rultor merge

**Comment by victornoel** [comment]
@llorllale I suppose the above is not meant to happen?

**Comment by llorllale** [comment]
@victornoel no... have you tried running gitlint on your local?

**Comment by victornoel** [comment]
@llorllale I fixed the commit ;)

**Comment by victornoel** [comment]
@llorllale I don't understand what is the problem, it is as if there was some utf8 characters somewhere, but I can't reproduce on my machine… also it seems that travis does not exhibit the problem, only rultor…

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
The job [#1029](https://github.com/yegor256/cactoos/issues/1029) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Pull request merged**